src/MentionsTextField.tsx

```diff
@@ -51,6 +51,13 @@ interface MentionsTextFieldBaseProps<T extends BaseSuggestionData> {
      * @default 'primary.light'
      */
     highlightColor?: string;
+
+    /**
+     * If true, mentions will be highlighted with text color (theme.palette.primary.main)
```

Comment:
I'm open to making this color customizable. Happy to hear your thoughts on this

Reply:
I think this can be a boolean, but the text should probably just use the color from `hightlightColor`. See my other comment.